src/main/java/com/class4/movieList/service/MovieListServiceImpl.java

```diff
@@ -11,125 +16,561 @@
+            HashMap<String, Object> peopleHashMap = mapper.readValue(strPeopleList, HashMap.class);
+            HashMap<String, Object> peopleListResult = (HashMap<String, Object>)peopleHashMap.get("peopleListResult");
+            
+            // peopleCd를 리스트에 보관
+            ArrayList<String> peopleCdList = new ArrayList<String>();
+            for(int i = 0; i < row; ++i) {
+               // peopleCd를 리스트에 추가
+               HashMap<String, Object> peopleList = (HashMap<String, Object>) ((ArrayList<Object>) peopleListResult.get("peopleList")).get(i);
+               peopleCdList.add(peopleList.get("peopleCd").toString());
+
+            }
+            
+            for(int i = 0; i < peopleCdList.size(); ++i) {
+               
+               // peopleCd로 영화인 정보 추출
+               String strActorInfo = service.getPeopleInfo(true, peopleCdList.get(i));
+               HashMap<String, Object> peopleResult = mapper.readValue(strActorInfo, HashMap.class);
+               LinkedHashMap<String, Object> peopleInfoResult = (LinkedHashMap<String, Object>) peopleResult.get("peopleInfoResult");
+               LinkedHashMap<String, Object> peopleInfo = (LinkedHashMap<String, Object>) peopleInfoResult.get("peopleInfo");
+               System.out.println(peopleInfo);
+               
+               // 인물코드, 이름, 영어이름
+               String peopleCd = peopleInfo.get("peopleCd").toString();
+               String peopleNm = peopleInfo.get("peopleNm").toString();
+               String peopleNmEn = peopleInfo.get("peopleNmEn").toString();
+               
+               
+               
+               // 기존 테이블에서 peopleCd 호출
+               ArrayList<String> tableActorCodeList = movieListMapper.getActorCodeList();
+               ArrayList<String> tableDirectorCodeList = movieListMapper.getDirectorCodeList();
+               
+               // 중복 여부 체크 변수
+               boolean actorFlag = true;
+               boolean directorFlag = true;
+               
+               for(int j = 0; j < tableActorCodeList.size(); ++j) {
+                  if(peopleCd.equals(tableActorCodeList.get(j))) {
+                     actorFlag = false;
+                     break;
+                  }
+               }
+               
+               for(int j = 0; j < tableDirectorCodeList.size(); ++j) {
+                  if(peopleCd.equals(tableDirectorCodeList.get(j))) {
+                     directorFlag = false;
+                     break;
+                  }
+               }
+               
+               System.out.println(1);
+               
+               // 중복 시 insert하지 않는다
+            
+               // 배우면 ActorVO에, 감독이면 DirectorVO에 추가
+               if(actorFlag && peopleInfo.get("repRoleNm").equals("배우")) {
+                  // Actor 테이블을 조회해서 이미 있는 peopleCd라면 추가하지 말고 출연, 감독한 영화로 이동
+                     
+                  ActorVO actorVO = new ActorVO(peopleCd, peopleNm, peopleNmEn);
+                  movieListMapper.ActorInsert(actorVO);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.